tests/features/common/kie-kieserver-common.feature

```diff
@@ -570,3 +570,37 @@ Feature: Kie Server common features
       | SCRIPT_DEBUG                           | true                          |
     Then container log should contain -Dorg.kie.kafka.server.ext.disabled=true
     And container log should contain Bootstrap servers not configured, kafka extension disabled
+
+  Scenario: Check if the Kafka JBPM Emitter is enabled
+    When container is started with env
+      | variable                                              | value                         |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_BOOTSTRAP_SERVERS | localhost:9093                |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED           | true                          |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID         | jbpmapp                       |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS              | 3                             |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS      | 2100                          |
+      | KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT       | dd-MM-yyyy'T'HH:mm:ss.SSSZ    |
+      | SCRIPT_DEBUG                                          | true                          |
+    Then container log should contain -Dorg.kie.kafka.server.ext.disabled=true
+    # Replace with the follow when this https://issues.redhat.com/browse/JBPM-9716 will be merged and the jar will be realigned
```

Comment:
jira is closed, is there anything else missing?

Reply:
No